services/core-api/app/api/mines/documents/resources/mine_document_version_resource.py

```diff
@@ -49,8 +57,16 @@ def post(self, mine_guid, mine_document_guid):
         if mine_document.is_archived:
             raise BadRequest('Cannot create new version of archived document')
 
-        return DocumentManagerService.initializeFileVersionUploadWithDocumentManager(
+        resp = DocumentManagerService.initializeFileVersionUploadWithDocumentManager(
             request, mine_document)
+        project_name = Project.find_by_project_guid(project_guid)
```

Comment:
Same for the above. What happens if this is a file not associated with a project?
Would be great to look up the associated project here instead of having to pass in the `project_guid`